config/samples/networking_v1alpha1_ratelimit.yaml

```diff
@@ -3,7 +3,15 @@ kind: RateLimit
 metadata:
   name: ratelimit-sample
 spec:
-  # Add fields here
-  ratelimitconfig:
-    Data:
-      field: test
+  targetRef:
```

Comment:
I think this target ref is too much complicated, something like KEDA would be nice:

```
  scaleTargetRef:
    deploymentName: noddle-vs
```
translated to
```
  rateTargetRef:
    vsName:  noddle-vs
```

This assumes, the target is always a Virtual serve and the namespace is always in local. Maybe we could add namespace as optional.

WDYT?

Reply:
what is vsName ? 
how can I ensure that the target is always Virtual service?